tests/test_standalone_stac_item.py

```diff
@@ -21,35 +20,11 @@ def quote_none_safe(url):
 
 @pytest.mark.online
 def test_standalone_stac_item_thredds_ncml():
-    thredds_url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds"
-    thredds_path = "birdhouse/testdata/xclim/cmip6"
-    thredds_nc = "sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
-    thredds_catalog = f"{thredds_url}/catalog/{thredds_path}/catalog.html"
-    thredds_ds = f"{thredds_path}/{thredds_nc}"
-    thredds_ncml_url = (
-        f"{thredds_url}/ncml/{thredds_path}/{thredds_nc}"
-        f"?catalog={quote_none_safe(thredds_catalog)}&dataset={quote_none_safe(thredds_ds)}"
-    )
-
-    # FIXME: avoid hackish workarounds
-    data = requests.get(thredds_ncml_url).text
-    attrs = xncml.Dataset.from_text(data).to_cf_dict()
-    attrs["access_urls"] = {  # FIXME: all following should be automatically added, but they are not!
-        "HTTPServer": f"{thredds_url}/fileServer/{thredds_path}/{thredds_nc}",
-        "OPENDAP": f"{thredds_url}/dodsC/{thredds_path}/{thredds_nc}",
-        "WCS": f"{thredds_url}/wcs/{thredds_path}/{thredds_nc}?service=WCS&version=1.0.0&request=GetCapabilities",
-        "WMS": f"{thredds_url}/wms/{thredds_path}/{thredds_nc}?service=WMS&version=1.3.0&request=GetCapabilities",
-        "NetcdfSubset": f"{thredds_url}/ncss/{thredds_path}/{thredds_nc}/dataset.html",
-    }
-
+    url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
```

Comment:
I suggest using `@pytest.mark.parametrize()` with the `url` as input. 

We could test the PAVICS URL as well as the https://psl.noaa.gov/thredds/catalog/Datasets/catalog.html that @dchandan referred to as problematic.

Reply:
parametrize: ok

But then we need a reference for a link that might change at any time, no ?

Reply:
Good point.
Instead, we could make a local copy of the XML content and leave a comment of the reference URL in case we need to look it up later.
Could be the same for the PAVICS URL as well.

Reply:
Done using responses.